draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1441,6 +1416,10 @@ as returned by `get-sth` in {{get-sth}}.
 
 The `start` parameter MUST be less than or equal to the `end` parameter.
 
+The `chain` field in the `submission` output parameter MUST include the trust
+anchor that the submission chained to, even if it was omitted in the original
```

Comment:
I suggest: s/that the submission chained to/that the log used to verify the submission/